src/render.js

```diff
@@ -11,9 +13,27 @@ import cheerio from 'cheerio';
  * thus I'd like to keep this API in here even though it's not really "ours".
  *
  * @param node
+ * @param options
  * @returns {Cheerio}
  */
-export default function render(node) {
+export default function render(node, options = {}) {
+  if (options.context && (node.type.contextTypes || options.childContextTypes)) {
+    const childContextTypes = node.type.contextTypes || {};
+    if (options.childContextTypes) {
+      objectAssign(childContextTypes, options.childContextTypes);
+    }
+    class ContextWrapper extends React.Component {
+      getChildContext() {
+        return options.context;
+      }
+      render() {
+        return node;
+      }
+    }
```

Comment:
Yeah that might be a little nicer, but I also think that it is okay the way it currently is written.